frontend-react/src/config/api/QueryClients.ts

```diff
@@ -0,0 +1,11 @@
+import { QueryClient } from "@tanstack/react-query";
+
+export const appQueryClient = new QueryClient({
```

Comment:
Just wanted to make sure [these changes](https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/network/QueryClients.ts#L3) are captured here!